ddtrace/internal/debug.py

```diff
@@ -88,6 +88,7 @@ def collect(tracer):
         agent_url = "http://%s:%s" % (hostname, port)
 
     if (hostname and port) or uds_path:
+        loc = "%s:%s" % (hostname, port) if hostname and port else uds_path
```

Comment:
could we start using the URL?

Reply:
definitely, I'll make a note to improve this when we stop using `API`